src/many-ledger/tests/common/mod.rs

```diff
@@ -1,22 +1,27 @@
-use std::{
```

Comment:
Let's avoid `mod.rs` files. There's no style guide for it really, but the Rust-lang team moved away from it and prefers having `x.rs` and `x/y.rs` instead of mod files.

cf https://github.com/rust-lang/rust/issues/53125 and https://doc.rust-lang.org/book/ch07-05-separating-modules-into-different-files.html#alternate-file-paths

Reply:
https://doc.rust-lang.org/book/ch11-03-test-organization.html#submodules-in-integration-tests

> To avoid having common appear in the test output, instead of creating tests/common.rs, we'll create tests/common/mod.rs. This is an alternate naming convention that Rust also understands. Naming the file this way tells Rust not to treat the common module as an integration test file. When we move the setup function code into tests/common/mod.rs and delete the tests/common.rs file, the section in the test output will no longer appear. Files in subdirectories of the tests directory don't get compiled as separate crates or have sections in the test output.

Reply:
Thanks.